daemon/containerd/image_tag.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 // TagImageWithReference creates an image named as newTag and targeting the given descriptor id.
-func (i *ImageService) TagImageWithReference(ctx context.Context, imageID image.ID, newTag reference.Named) error {
+func (i *ImageService) TagImageWithReference(ctx context.Context, imageID image.ID, newTag reference.Named) (retErr error) {
```

Comment:
I don't see any special handling (defer, e.g.) for the returned error, so we probably don't need a named `retErr` here.

Reply:
Ah good catch, I used it at first for some defer block, but then removed it and forgot to remove the named return. Fixed, thanks!